### ISSUE TITLE
Update to swift 4.2

### ISSUE THREAD

**Issue by codecat15** [evermeer/EVReflection#297]
I updated my project from swift 3 to swift 4.2 where we observed the given error

<img width="384" alt="screen shot 2018-11-04 at 11 26 47 am" src="https://user-images.githubusercontent.com/44252456/47967603-aa92ec00-e024-11e8-993c-ee7438935d8b.png">

In my build settings i did add the user defined settings as suggested by apple 

SWIFT_ENABLE_BATHC_MODE and set it to NO but even after that i am getting this error, should i use an older version or am i missing a configuration setting here please suggest.

**Comment by evermeer** [comment]
After converting to 4.2, did you do a 'pod update'?

**Comment by codecat15** [comment]
@evermeer I got this error when I tried to upgrade from swift 4 to 4.2, I reverted my pod project back to 4.0 as I was still getting the error after a `pod update` so I repeat the below steps because for some reason pod update was not working for me.

1. Delete the pod EVReflection from pod file
2. Do a pod install
3. Do a clean build and rebuild the project
4. Here I get a build success message and after that, I re-add `pod EVReflection` to my pod file
5. repeat step no 2

And now the error is now gone, I verified the version of EVReflection and it's showing as `5.8.0` which is the latest version.
